### DISCUSSION TITLE
Any plans to migrate to Mover2.0?

### DISCUSSION THREAD

**Discussion by jjonj** [Sixze/ALS-Refactored#465] [Ideas]
It seems Mover2.0 is the way of the future in unreal locomotion, especially for multiplayer games.

Do you have any plans to upgrade to the project to Mover2.0 eventually?

Thanks for all your hard work!

**Comment by xkamronx** [comment]
I believe the ALS movement component is basically completely custom, not based on the default CMC, so I'd think changing it over to Mover2 would basically be rewriting the entirety of it?

**Reply by murataka** [reply]
Als extends the movement component, does not reimplement it from scratch. I foresee that will be a massive improvement to ALS, with minimal effort .

**Reply by xkamronx** [reply]
I stand corrected :) Thank you for the info!

**Comment by Magic-Summer** [comment]
According to the author, there are currently no [plans](https://github.com/Sixze/ALS-Refactored/discussions/424#discussioncomment-7951561) to migrate to Mover2.0.
However, there are plans to integrate GAS.
